sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1204,10 +1204,14 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "programYear.startDate",
         "user.userName",
         "student.id",
+        "assessment.id",
+        "offering.offeringIntensity",
       ])
       .innerJoin("application.student", "student")
       .innerJoin("student.user", "user")
       .innerJoin("application.programYear", "programYear")
+      .innerJoin("application.currentAssessment", "assessment")
```

Comment:
During our sync up, I forgot to evaluate one scenario of application with PIR and Partner and partner  trying to submit application before PIR is done. 

In this case offering is still not assigned to the application and the logic will fail. Hence we need to take the application data input for the offering intensity here.

Reply:
Thanks for doing the change.